hbase-client/src/main/java/org/apache/hadoop/hbase/client/Hbck.java

```diff
@@ -105,8 +109,18 @@
   List<Boolean> bypassProcedure(List<Long> pids, long waitTime, boolean override, boolean recursive)
       throws IOException;
 
-  List<Long> scheduleServerCrashProcedure(List<HBaseProtos.ServerName> serverNames)
-      throws IOException;
+  /**
+   * Use {@link #scheduleServerCrashProcedures(List)} instead.
+   * @deprecated since 2.2.1. Will removed in 2.3.0.
```

Comment:
Can you please create a ticket targeting the 2.3.0 release and the current master for removal?

Reply:
Open a new issue HBASE-22708 for it.